CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.1.1] - TBD
+### Changed
+- Updated Jenkinsfile to use correct agent.
```

Comment:
You don't need to document this here, end users don't care ;)

Reply:
Fair enough, done!